.travis.yml

```diff
@@ -7,7 +7,7 @@ go:
 install: true
 
 before_install:
-  go get github.com/codegangsta/cli && go get github.com/stretchr/testify/assert
+  bin/deps
```

Comment:
Yeah, probably. I'll pull this in and then tweak that in a branch to make sure it works.